packages/angular/src/app/components/contact-new-user-form/contact-new-user-form.component.ts

```diff
@@ -23,7 +24,7 @@ import { newContact } from 'src/app/types/contact';
 export class ContactNewUserFormComponent {
   newUser = newContact;
 
-  constructor() { }
+  constructor(protected screen: ScreenService) { }
```

Comment:
Why the service is protected?

Reply:
it's used in template, so it can't be private